client/components/editor/Sidebar/index.vue

```diff
@@ -91,6 +87,14 @@ export default {
     metadata() {
       const { repository, selectedElement } = this;
       return getElementMetadata(get(repository, 'schema'), selectedElement);
+    },
+    toolbarClasses() {
+      const { selectedElement } = this;
+      if (!selectedElement) return;
+
+      const classes = ['toolbar-visible'];
+      if (selectedElement.parent) classes.push('toolbar-composite');
```

Comment:
I do not know when the `toolbar-composite` class is applied. By that, I mean which content elements trigger it. If anybody knows please, write in a comment.

Reply:
Table

Reply:
I forgot about the table element 😄 . I'll test with it as well.